client/js/verification-portal/form.js

```diff
@@ -292,6 +301,13 @@ const styles = {
 	},
 	form: {
 		position: 'relative',
+		display: 'flex',
+		'flex-direction': 'column',
+		'align-items': 'baseline',
+		'justify-content': 'space-evenly',
+	},
+	formSection: {
+		width: '100%',
```

Comment:
There is no functionality change in this file. I just added flexbox and partitioned the form into sections so that it aligns better. With the formatting changes I made to the checkbox, it ended up messing up the form quite a bit.